server/src/test/java/org/elasticsearch/cluster/metadata/IndexMetadataTests.java

```diff
@@ -82,6 +83,15 @@ public void testIndexMetadataSerialization() throws IOException {
         IndexMetadataStats indexStats = randomBoolean() ? randomIndexStats(numShard) : null;
         Double indexWriteLoadForecast = randomBoolean() ? randomDoubleBetween(0.0, 128, true) : null;
         Long shardSizeInBytesForecast = randomBoolean() ? randomLongBetween(1024, 10240) : null;
+        Map<String, Set<String>> fieldsForModels = new HashMap<>();
```

Comment:
Nice! 👍

Reply:
Thanks! I think this random `fieldsForModels` generation logic is large enough that I should refactor it into a helper method.